pubtools/_quay/command_executor.py

```diff
@@ -78,7 +79,7 @@ def tag_images(self, source_ref, dest_refs, all_arch=False):
             LOG.info(
                 "Tagging source '{0}' to destination '{1}'".format(source_ref, dest_ref)
             )
-            self._run_cmd(cmd.format(source_ref, dest_ref))
+            self._run_cmd(cmd.format(pipes.quote(source_ref), pipes.quote(dest_ref)))
```

Comment:
Would it be possible to move the sanitization to RemoteExecutor's _run_cmd? That way we won't have to think about it when we add new commands.

Reply:
The quote needs to be applied to user input. RemoteExecutor gets the whole command string so which part is from input can't be identified.